run.py

```diff
@@ -253,6 +261,15 @@ def internal_server_error_handler(e):
         layout = LayoutHelper(logged_in=False)
     return render_template('500.html', layout=layout.get_entries()), 500
 
+@app.errorhandler(503)
+def unauthorized_handler(e):
```

Comment:
This should not be `unauthorized_handler`. 503 is `Service Unavailable`.

Reply:
Just fixed it in 5d60b74